packages/web/src/scss/components/Icon/README.md

```diff
@@ -0,0 +1,37 @@
+# Icon
+
+## Basic Usage
+
+```html
+<svg width="24" height="24">
+  <use xlink:href="/assets/icons/svg/sprite.svg#info" />
+</svg>
+```
+
+## Responsive Size
+
+Use CSS custom properties to define the `width` and `height` of the icon.
+
+- `--spirit-icon-width: number{px};`
```

Comment:
do we want two props? Maybe one would be enough? To cover both width and height.